_posts/2017-07-29-css-grids.md

```diff
@@ -0,0 +1,170 @@
+---
+layout: post
+author: Cyan/Jos
+title: Grid CSS
+---
+
+### What is CSS Grids?
+CSS Grids is a new standard in CSS3 and is slowly being adopted by all web browsers.
+Creating a basic website like this one requires a few lines of CSS, for example:
+
+![Example Site]({{ site.url }}/assets/css-grids/example-site.png)
+
+And the CSS:
+
+```css
+.wrap {
+    border: 5px solid red;
+    height: 400px;
+}
+
+.main-content {
+    width: 45%;
+    margin-left: 1%;
+    height: 98%;
+    float: left;
+    background-color: aqua;
+}
+
+.sidebar {
+    width: 45%;
+    margin-right: 1%;
+    height: 98%;
+    float: right;
+    background-color: crimson;
+}
+```
+
+---
+
+Grid creates, well, a grid inside a div element (Think of lines between the space in the image below). 
```

Comment:
> Grid creates, well,

Try to avoid the speech in-formalities and take a more instructional tone. (`-well,`)